Tools/jit/_targets.py

```diff
@@ -212,13 +212,17 @@ def build(
         ):
             return
         stencil_groups = asyncio.run(self._build_stencils())
-        with jit_stencils.open("w") as file:
-            file.write(digest)
-            if comment:
-                file.write(f"// {comment}\n\n")
-            file.write("")
-            for line in _writer.dump(stencil_groups):
-                file.write(f"{line}\n")
+        jit_stencils_new = out / "jit_stencils.h.new"
```

Comment:
Is there a reason you chose to add `new` as part of the file extension instead of doing something like `new_jit_stencils.h`? 

It probably doesn't matter much since this just ends up being used to overwrite the stencils later but just curious!

Reply:
Mostly just convention. Search our Makefile for ".new"... you'll see that this is a common pattern for generated files!